datagrid_gtk3/db/sqlite.py

```diff
@@ -143,6 +154,8 @@ def __init__(self, db_file, table=None, update_table=None, config=None,
                                    'RENAME TO __visible_columns')
                     conn.commit()
 
+        self.__class__._DBS[(db_file, self.table.name, id(self))] = self
```

Comment:
From what I see below the `_DBS` class attribute is always traversed using the `items` method, so the key is unnecessary since both the db file and the table name can be retrieved from the object that is stored as value. What about using `weakref.WeakSet` instead?